### PR TITLE
msal-node e2e tests: cert_path -> certificate_path

### PR DESCRIPTION
**Pull request by Robbie-Microsoft**
@azure/identity's `new DefaultAzureCredential()` expects `AZURE_CLIENT_CERTIFICATE_PATH` instead of `AZURE_CLIENT_CERT_PATH`.

[Corresponding 1-P PR](https://identitydivision.visualstudio.com/IDDP/_git/msal-javascript-1p/pullrequest/14791)

### CODE REVIEW

File: .pipelines/3p-e2e.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ resources:
         - repository: 1P
           type: git
           name: IDDP/msal-javascript-1p
-          ref: master
+          ref: keyvault_cert_sample
```
Comment (Robbie-Microsoft) [inline .pipelines/3p-e2e.yml]: This change is made for testing against the corresponding 1P change. This file will be reverted before the PR is merged into dev.

**Comment by hectormmg** [comment]
Can you rename your branch so it's the same as the 1P branch? Otherwise the pipelines will not checkout the right code.

File: .pipelines/3p-e2e.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ resources:
         - repository: 1P
           type: git
           name: IDDP/msal-javascript-1p
-          ref: master
+          ref: rename_cert_path
```
Comment (Robbie-Microsoft) [inline .pipelines/3p-e2e.yml]: This ref and the rest of this file's changes will be reverted before merging into dev. This ref change is needed for pointing to the corresponding 1P change.

File: gen_env.ps1 [hunk 3]
```diff
@@ -2,7 +2,7 @@
 $dotEnvFileName = ".env"
 $tenantIdInfo = 'AZURE_TENANT_ID="72f988bf-86f1-41af-91ab-2d7cd011db47"'
 $clientIdName = "AZURE_CLIENT_ID="
-$clientCertPathName = "AZURE_CLIENT_CERT_PATH="
+$clientCertPathName = "AZURE_CLIENT_CERTIFICATE_PATH="
```
Comment (bgavrilMS) [inline gen_env.ps1:7]: Pls add a comment linking to Azure Identity's docs - https://learn.microsoft.com/en-us/dotnet/api/azure.identity.environmentcredential?view=azure-dotnet i.e. this env var name has to be exactly that.

**Review by bgavrilMS — APPROVED** [review]

**Review by konstantin-msft — APPROVED** [review]

**Review by tnorling — APPROVED** [review]

File: gen_env.ps1 [hunk 4]
```diff
@@ -2,7 +2,7 @@
...
 $sessionSecretName = "SESSION_SECRET="
```
Comment (Robbie-Microsoft) [inline gen_env.ps1:9]: CC @hectormmg I'm not sure when/why this file gets run. Should I add `$AZURE_CLIENT_SEND_CERTIFICATE_CHAIN = true` here? Or just leave that on the 1P PR.

Comment (hectormmg) [reply gen_env.ps1:9]: This is a developer script, it's run by us manually when setting up the repo. You can delete your .env and run this to make sure it creates the right .env file with the right variable name

**Review by sameerag — APPROVED** [review]

**Pull request merged**